src/main/resources/hibernate.cfg.xml

```diff
@@ -13,6 +13,10 @@
         <property name="show_sql">true</property>
         <property name="hbm2ddl.auto">update</property>
 
-
+        <mapping class="core.basesyntax.model.Comment"/>
+        <mapping class="core.basesyntax.model.Message"/>
+        <mapping class="core.basesyntax.model.MessageDetails"/>
+        <mapping class="core.basesyntax.model.Smile"/>
+        <mapping class="core.basesyntax.model.User"/>
```

Comment:
Maybe it's optional our application works without mapping. And in common mistakes was said that we don't need to change any file except models and daoImpl